src/components/reader/DoublePage.tsx

```diff
@@ -68,10 +56,12 @@ export const DoublePage = forwardRef((props: IProps, ref: any) => {
                 imgRef={imgRef}
                 spinnerStyle={{
                     ...spinnerStyle,
-                    width: 'calc((100vw - 300px - 5px) * 0.5)',
                     marginLeft: '5px',
                 }}
-                imgStyle={imgStyles}
+                imgStyle={{
+                    ...imgStyle,
+                    marginLeft: '5px',
```

Comment:
the pages are not supposed to have a gap

Reply:
Oh, so you're right. I think I thought a gap was needed because of other changes I made that added one.

Reply:
the horizontal pager is fixed but gap in the double pager is still there

Reply:
Ok, I think that should be fixed now.